policy-controller/k8s/index/src/outbound/index.rs

```diff
@@ -435,71 +440,65 @@ impl ServiceRoutes {
-                .map(|s| {
-                    s.parse::<u32>()
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                }) // Option<Result<>
-                .transpose()?
-                .unwrap_or(7);
-
-            let max_penalty = annotations
-                .get("balancer.linkerd.io/max-penalty")
-                .and_then(|s| {
-                    parse_duration(s)
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                })
-                .unwrap_or_else(|| time::Duration::from_secs(60));
-
-            let min_penalty = annotations
-                .get("balancer.linkerd.io/min-penalty")
-                .and_then(|s| {
-                    parse_duration(s)
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                        .ok()
-                })?
-                .unwrap_or_else(|| time::Duration::from_secs(1));
-            let jitter = annotations
-                .get("balancer.linkerd.io/jitter")
-                .and_then(|s| {
-                    s.parse::<f64>()
-                        .map_err(|error| tracing::debug!(%error, "Invalid annotation value"))
-                        .ok()
-                })
-                .unwrap_or_default();
-            if min_penalty > max_penalty {
-                tracing::debug!(?min_penalty, ?max_penalty, "Invalid backoff configuration");
-                return Some(FailureAccrual::Consecutive {
+    annotations
+        .get("balancer.linkerd.io/mode")
+        .map(|mode| {
+            if mode == "consecutive" {
+                let max_failures = annotations
+                    .get("balancer.linkerd.io/failure-accrual-consecutive-failures")
+                    .map(|s| s.parse::<u32>())
+                    .transpose()?
+                    .unwrap_or(7);
+
+                let max_penalty = annotations
+                    .get("balancer.linkerd.io/max-penalty")
+                    .map(|s| parse_duration(s))
+                    .transpose()?
+                    .unwrap_or_else(|| time::Duration::from_secs(60));
+
+                let min_penalty = annotations
+                    .get("balancer.linkerd.io/min-penalty")
+                    .map(|s| parse_duration(s))
+                    .transpose()?
+                    .unwrap_or_else(|| time::Duration::from_secs(1));
+                let jitter = annotations
+                    .get("balancer.linkerd.io/jitter")
+                    .map(|s| s.parse::<f64>())
+                    .transpose()?
+                    .unwrap_or_default();
+                if min_penalty > max_penalty {
+                    bail!(
+                        "min_penalty ({:?}) cannot exceed max_penalty ({:?})",
+                        min_penalty,
+                        max_penalty,
+                    );
```

Comment:
style nit, take it or leave it: could be
```suggestion
                    bail!("min_penalty ({min_penalty:?}) cannot exceed max_penalty ({max_penalty:?})");
```
(assuming rustfmt doesn't want to wrap it)